app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -387,7 +387,12 @@ class BrowserWebViewClient @Inject constructor(
         }
 
         Timber.d("The certificate authority validation result is $trusted")
-        if (trusted is CertificateValidationState.TrustedChain) handler.proceed() else super.onReceivedSslError(view, handler, error)
+        if (trusted is CertificateValidationState.TrustedChain) {
+            handler.proceed()
+        } else {
+            webViewClientListener?.onReceivedError(WebViewErrorResponse.SSL_PROTOCOL_ERROR, view?.url.toString())
```

Comment:
Added this line to show the error view and left untouched the default behavior to cancel the load done by `super.onReceivedSslError(view, handler, error)`.